### PR TITLE
Fix cluster dashboard opening when cluster is initally offline

### LINKED ISSUE #1035: Blank page after minikube started
**Describe the bug**
This is an easy way to reproduce a blank page. It happens when Lens is started and then minikube is (re)started

**To Reproduce**
1. add a minikube cluster in Lens
2. quit Lens
3. run minikube stop
4. start Lens (you'll see Lens tries unsuccessfully to connect--this is ok, expected)
5. run minikube start
6. After minikube starts observe the connection error is replaced by a blank page

**Expected behavior**
Cluster page should be shown.

**Environment (please complete the following information):**
- Lens Version: 3.6.6
- OS: linux and MacOS
- Installation method (e.g. snap or AppImage in Linux): AppImage and dmg

**Additional context**
Lens behaves properly if, while Lens is running, you stop minikube and then start minikube again. It seems to only be a problem when Lens starts up after minikube is stopped

### PR DESCRIPTION
**Pull request by nevalla**
Fixes #1035 

Signed-off-by: Lauri Nevala <lauri.nevala@gmail.com>

### CODE REVIEW

File: src/main/cluster.ts [hunk 1]
```diff
@@ -234,7 +237,7 @@ export class Cluster implements ClusterModel {
     const apiUrl = this.kubeProxyUrl + path;
     return request(apiUrl, {
       json: true,
-      timeout: 5000,
+      timeout: 30000,
```
Comment (nevalla) [inline src/main/cluster.ts:240]: Increasing timeout just to be on the safe side.

File: src/renderer/components/cluster-manager/cluster-view.tsx [hunk 2]
```diff
@@ -9,7 +9,7 @@ import { hasLoadedView } from "./lens-views";
 export class ClusterView extends React.Component {
   render() {
     const cluster = getMatchedCluster();
-    const showStatus = cluster && (!cluster.available || !hasLoadedView(cluster.id))
+    const showStatus = cluster && (!cluster.available || !hasLoadedView(cluster.id) || !cluster.ready)
```
Comment (nevalla) [inline src/renderer/components/cluster-manager/cluster-view.tsx:12]: Show the cluster status until the cluster is ready.

**Review by aleksfront — APPROVED** [review]

File: src/main/cluster.ts [hunk 3]
```diff
@@ -184,6 +184,9 @@ export class Cluster implements ClusterModel {
         this.refreshEvents(),
         this.refreshAllowedResources(),
       ]);
+      if (!this.ready) {
```
Comment (Nokel81) [inline src/main/cluster.ts:187]: Shouldn't this have been just `this.ready = true`?

Comment (nevalla) [reply src/main/cluster.ts:187]: Yes...should be fine.

Comment (Nokel81) [reply src/main/cluster.ts:187]: It should be. I just saw this as I was going through older PRs to know what was changed. I guess if `ready` was in fact `set ready() { ... }` with a side effect then we may not want to always set it. But I think in this case we can change it later.

**Pull request merged**